schema.py

```diff
@@ -0,0 +1,40 @@
+import graphene
+from graphene_sqlalchemy import SQLAlchemyObjectType, SQLAlchemyConnectionField
+from graphene import relay, Schema, ObjectType
+from models import User, Story,db_session
+
+
+class UserType(SQLAlchemyObjectType):   
+    class Meta:        
+        model = User        
+        interfaces = (relay.Node,)
+
+class StoryType(SQLAlchemyObjectType):    
+    class Meta:        
+        model = User       
+        interfaces = (relay.Node,)
+
+
+class UpdateUsername(graphene.Mutation):
+	class Arguments:
+		username = graphene.String()
+		email = graphene.String()
+
+	user = graphene.Field(UserType)
+
+	def mutate(self,info,email,username):   
```

Comment:
This mutate method restrits you to only change the username, what would be the case if you wanted to modify first_name, last_name,, password etc, consider checking all supplied fields and only modified those that have changed